packages/eds-core-react/src/components/Button/Button.stories.tsx

```diff
@@ -139,46 +178,126 @@ Hierarchy.decorators = [
   ),
 ]
 
-export const FileUpload: Story<ButtonProps> = () => (
-  <label htmlFor="file-upload">
-    <input type="file" id="file-upload" style={{ display: 'none' }} multiple />
-    <Button as="span">Upload file</Button>
-  </label>
-)
-FileUpload.storyName = 'File upload'
-FileUpload.decorators = [
+export const Compact: Story<ButtonProps> = () => {
+  const [compact, setComfortable] = useState<boolean>(true)
+
+  return (
+    <EdsProvider density={compact ? 'compact' : 'comfortable'}>
+      <Checkbox
+        label="Compact"
+        onChange={() => {
+          setComfortable(!compact)
+        }}
+        checked={compact}
+      />
+      <Stack direction="row">
+        <Button>Contained</Button>
+        <Button variant="outlined">Outlined</Button>
+        <Button variant="ghost">Ghost</Button>
+        <Button variant="ghost_icon" aria-label="menu action">
+          <Icon data={menu} title="Ghost icon menu"></Icon>
+        </Button>
+      </Stack>
+    </EdsProvider>
+  )
+}
+Compact.decorators = [
+  (Story) => (
+    <Stack direction="column">
+      <Story />
+    </Stack>
+  ),
+]
+
+export const ProgressButton: Story<ButtonProps> = () => {
+  const [isSubmitting, setIsSubmitting] = useState<boolean>(false)
+
+  const onSubmit = () => {
+    setIsSubmitting(true)
+  }
+
+  return (
+    <>
+      <Button
+        aria-disabled={isSubmitting ? true : false}
+        onClick={!isSubmitting ? onSubmit : undefined}
+      >
+        {isSubmitting ? <Progress.Dots /> : 'Fetch data'}
+      </Button>
+      <Button
+        aria-disabled={isSubmitting ? true : false}
+        color="secondary"
+        onClick={!isSubmitting ? onSubmit : undefined}
+      >
+        {isSubmitting ? (
+          <Progress.Circular size={16} color="neutral" />
+        ) : (
+          <>
+            Send
+            <Icon data={send} size={16}></Icon>
+          </>
+        )}
+      </Button>
+      <Button onClick={() => setIsSubmitting(false)}>
+        <>
+          <Icon data={refresh} size={16}></Icon>
+          Reset
+        </>
+      </Button>
+    </>
+  )
+}
+
+ProgressButton.storyName = 'Progress button'
+ProgressButton.decorators = [
   (Story) => (
     <Stack>
       <Story />
     </Stack>
   ),
 ]
 
-export const ProgressButton: Story<ButtonProps> = () => (
+export const WithTooltip: Story<ButtonProps> = () => (
   <>
-    <Button>
-      <Progress.Dots />
-    </Button>
-    <Button variant="ghost_icon">
-      <Progress.Dots color="primary" />
-    </Button>
-    <Button>
-      <Progress.Circular size={16} color="neutral" />
-    </Button>
-    <Button variant="ghost_icon">
-      <Progress.Circular size={16} color="primary" />
-    </Button>
+    <Tooltip title="This is what a tooltip looks like">
+      <Button>Hover me</Button>
+    </Tooltip>
+    <Tooltip title="Tooltip doesn't show in Chrome">
```

Comment:
can we change text to: "This tooltip only shows for people using firefox and using mouse. Don't do this!"